contract/src/main.rs

```diff
@@ -916,20 +903,23 @@ pub extern "C" fn set_approval_for_all() {
         }
         let operator = if approve_all { Some(operator) } else { None };
         storage::dictionary_put(operator_uref, &token_id.get_dictionary_item_key(), operator);
+        let events_mode: u8 = get_stored_value_with_user_errors(
+            crate::constants::EVENTS_MODE,
+            NFTCoreError::MissingEventsMode,
+            NFTCoreError::InvalidEventsMode,
+        );
+        let events_mode = EventsMode::try_from(events_mode).unwrap_or_revert();
+        if events_mode != EventsMode::NoEvents {
+            events::record_event(match events_mode {
+                // EventsMode::CEP47 => Event::Cep47(CEP47Event::ApproveAll {
```

Comment:
Commented code here

Reply:
Thanks I made corresponding change and Zoltan will add a test 

https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/143/commits/50bdd038caaa9e95445c246fede42d762c3f3662